### PR TITLE
Grayscaled Alarm and Status Display Icons

### PR DESCRIPTION
**Pull request by Mazianni**
This PR changes fire alarms and status displays to using greyscaled icons instead of static icons. Included are a handful I whipped up, so this should allow for an easier variety of code displays.

### CODE REVIEW

File: code/datums/security_state.dm [hunk 1]
```diff
@@ -148,6 +148,13 @@
 	var/up_description
 	var/down_description
 
+	var/datum/alarm_appearance/alarm_appearance
+	var/alarm_appearance_type
+
+/decl/security_level/New()
+	. = ..()
+	alarm_appearance = new alarm_appearance_type
```
Comment (MistakeNot4892) [inline code/datums/security_state.dm]: Need to typecheck this in case it's null, since the default value is null.

**Review by MistakeNot4892 — CHANGES_REQUESTED** [review]
Please rebase and squash this, then I will approve it for merge.

**Review by MistakeNot4892 — APPROVED** [review]
If I was a cruel man I'd go :eyes: at the fix commit.

**Comment by Mazianni** [comment]
🔪

**Pull request merged**